CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+## Pull requests and Code contributions
+
+* Tests must pass.
+* Follow our coding style (jscs and jshint will help you).
+* If you fix a bug, add a test.
+* If you can't fix a bug, file an [issue](https://github.com/gemini-testing/hermione/issues) with the steps to reproduce, the expected and the actual results.
+* if an issue desribes a problem which we are unable to reproduce and does not answer our questions for 1 week then this issue may be closed without further investigation.
```

Comment:
if an issue desribes a problem which we are unable to reproduce and `issue reporter` does not answer our questions..